client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -46,7 +46,7 @@ module.exports = React.createClass( {
 
 	render() {
 		return (
-			<Main className="notifications">
+			<Main className="notifications-settings">
```

Comment:
Sorry for nitpicking, but should be `notification-settings` or `reader-subscriptions` :)